api/engine.py

```diff
@@ -41,6 +46,11 @@
 from database.db import __logs_to_report_json
 from database.db import __search_logs
 from database.db import __logs_to_report_html
+from api.__start_scan import __scan
+from core._time import now
```

Comment:
@aman566  do you really need this `now`? there is `datetime.now` beware of redeclaration issue please check flake8/pylint results